### PR TITLE
Updated files to take input at runtime

### LINKED ISSUE #1179: Updating files to take input at runtime (#1160)
Files updating: 
1.Bubble_Sort.c
2.Bubble_Sort.cpp
3.Counting_Sort.c
4.Counting_Sort.cpp
5.Heap_Sort.c
6.Heap_Sort.cpp

### PR DESCRIPTION
**Pull request by somya-kapoor**
Partly solves #1160 
Fixes #1179 
#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Updated files to take input at runtime

#### Languages Used:
- C
- C++

#### Files Added: (Partly solves)
- Bubble_Sort.c 
- Bubble_Sort.cpp
- Counting_Sort.c
- Counting_Sort.cpp
- Heap_Sort.c
- Heap_Sort.cpp

### CODE REVIEW

**Comment by MastersAbh** [comment]
Hi,
The issue is still unassigned. You must wait for it to be assigned to you.

**Comment by jainaman224** [comment]
Please create another issue referencing main issue. So that we can keep track of what is fixed.

**Comment by somya-kapoor** [comment]
@jainaman224 Please review.

**Comment by jainaman224** [comment]
Please squash your commits.

**Comment by somya-kapoor** [comment]
@jainaman224 Done, Please review.

**Comment by somya-kapoor** [comment]
@jainaman224 Done, Please review.

**Comment by somya-kapoor** [comment]
@jainaman224 Done, Please review.

File: Bubble_Sort/Bubble_Sort.c [hunk 1]
```diff
@@ -36,11 +36,26 @@ void Print_Array(int array[], int size)
...
 {
-    int array[] = {2, 4, 3, 1, 6, 8, 4};
-    Bubble_Sort(array, 7);
-    Print_Array(array, 7);
+    int n;
+    printf("Enter number of elements in your array: ");
+    scanf("%d", &n);
+    int array[n];
+    printf("Enter your array: ");
+    for(int i = 0; i < n; i++)
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.c]: space after for

File: Bubble_Sort/Bubble_Sort.cpp [hunk 2]
```diff
@@ -40,11 +40,28 @@ void Print_Array(int array[], int size)
...
 {
-    int array[] = {2, 4, 3, 1, 6, 8, 4};
-    Bubble_Sort(array, 7);
-    Print_Array(array, 7);
+    int num;
+    cout << "Enter number of elements in your array: ";
+    cin  >> num;
+    int array[num];
+    cout << "Enter your array: ";
+    for(i = 0; i < num; i++)
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.cpp]: space after for

**Comment by somya-kapoor** [comment]
@MastersAbh Done please review.

**Comment by somya-kapoor** [comment]
@jainaman224 Done, Please review.

File: Bubble_Sort/Bubble_Sort.cpp [hunk 3]
```diff
@@ -40,11 +40,28 @@ void Print_Array(int array[], int size)
 // Driver Function
 int main()
 {
-    int array[] = {2, 4, 3, 1, 6, 8, 4};
-    Bubble_Sort(array, 7);
-    Print_Array(array, 7);
+    int num;
+    cout << "Enter number of elements in your array: ";
+    cin  >> num;
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.cpp]: please remove spaces around << & >>

File: Counting_Sort/Counting_Sort.c [hunk 4]
```diff
@@ -46,9 +46,14 @@ void counting_sort(int input[], int n)
...
 {
-    int n = 9, i;
+    int n, i;
+    printf("Enter number of elements in your array: ");
+    scanf("%d", &n);
 
-    int input[] = {1, 5, 2, 7, 3, 4, 4, 1, 5};
+    int input[n];
+    printf("Enter your array: ");
+    for(int i = 0; i < n; i++)
```
Comment (MastersAbh) [inline Counting_Sort/Counting_Sort.c]: space after for

File: Counting_Sort/Counting_Sort.cpp [hunk 5]
```diff
@@ -50,9 +50,16 @@ void counting_sort(int input[], int n)
...
 {
-    int n = 9, i;
+    int n, i;
+    cout << "Enter number of elements in your array: ";
+    cin  >> n;
 
-    int input[] = {1, 5, 2, 7, 3, 4, 4, 1, 5};
+    int input[n];
+    cout << "Enter your array: ";
+    for(i = 0; i < n; i++)
```
Comment (MastersAbh) [inline Counting_Sort/Counting_Sort.cpp]: space after for

File: Counting_Sort/Counting_Sort.cpp [hunk 6]
```diff
@@ -50,9 +50,16 @@ void counting_sort(int input[], int n)
...
+    {
+        cin  >> input[i];
+    }
```
Comment (MastersAbh) [inline Counting_Sort/Counting_Sort.cpp]: remove spaces around << & >>

**Comment by somya-kapoor** [comment]
@MastersAbh Done please review.

**Comment by somya-kapoor** [comment]
@jainaman224 Done, Please review.

File: Bubble_Sort/Bubble_Sort.cpp [hunk 7]
```diff
@@ -40,11 +40,28 @@ void Print_Array(int array[], int size)
...
 {
-    int array[] = {2, 4, 3, 1, 6, 8, 4};
-    Bubble_Sort(array, 7);
-    Print_Array(array, 7);
+    int num;
+    cout<<"Enter number of elements in your array: ";
+    cin>>num;
+    int array[num];
+    cou<<"Enter your array: ";
+    for (i = 0; i < num; i++)
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.cpp]: Throws error. i not declared.

File: Counting_Sort/Counting_Sort.c [hunk 8]
```diff
@@ -46,20 +46,33 @@ void counting_sort(int input[], int n)
...
 {
-    int n = 9, i;
+    int n, i;
+    printf("Enter number of elements in your array: ");
+    scanf("%d", &n);
 
-    int input[] = {1, 5, 2, 7, 3, 4, 4, 1, 5};
+    int input[n];
+    printf("Enter your array: ");
+    for (int i = 0; i < n; i++)
```
Comment (MastersAbh) [inline Counting_Sort/Counting_Sort.c]: please remove redeclaration of i. It's already declared outside the scope.

File: Heap_Sort/Heap_Sort.c [hunk 9]
```diff
@@ -58,11 +58,24 @@ void Print_Array(int array[], int size)
...
 {
-    int array[] = {10, 2, 4, 3, 1, 6, 8, 5};
-    Heap_Sort(array, 7);
-    Print_Array(array, 7);
+    int n, i;
+    printf("Enter number of elements in your array: ");
+    scanf("%d", &n);
+    int array[n];
+    printf("Enter your array: ");
+    for(int i = 0; i < n; i++)
```
Comment (MastersAbh) [inline Heap_Sort/Heap_Sort.c]: redeclaration of i.

Comment (MastersAbh) [reply Heap_Sort/Heap_Sort.c]: space after for

File: Heap_Sort/Heap_Sort.cpp [hunk 10]
```diff
@@ -55,11 +55,26 @@ void Print_Array(int array[], int size)
 
 int main()
 {
-    int array[] = {2, 4, 3, 1, 6, 8, 4};
-    Heap_Sort(array, 7);
-    Print_Array(array, 7);
+    int n, i;
+    cout << "Enter number of elements in your array: ";
+    cin  >> n;
```
Comment (MastersAbh) [inline Heap_Sort/Heap_Sort.cpp]: please remove spaces around << and >>

**Comment by somya-kapoor** [comment]
@MastersAbh Done please review.

File: Heap_Sort/Heap_Sort.cpp [hunk 11]
```diff
@@ -55,11 +55,26 @@ void Print_Array(int array[], int size)
...
 int main()
 {
-    int array[] = {2, 4, 3, 1, 6, 8, 4};
-    Heap_Sort(array, 7);
-    Print_Array(array, 7);
+    int n, i;
+    cout<<"Enter number of elements in your array: ";
+    cin>>n;
+    int array[n];
+    cout << "Enter your array: ";
```
Comment (MastersAbh) [inline Heap_Sort/Heap_Sort.cpp]: please remove spaces around <<

File: Heap_Sort/Heap_Sort.cpp [hunk 12]
```diff
@@ -55,11 +55,26 @@ void Print_Array(int array[], int size)
...
+    for(i = 0; i < n; i++)
```
Comment (MastersAbh) [inline Heap_Sort/Heap_Sort.cpp]: space after for

**Comment by somya-kapoor** [comment]
@MastersAbh Done please review.

**Pull request merged**